ocdata/utils/vasp.py

```diff
@@ -57,7 +63,7 @@ def run_vasp(atoms, vasp_flags=None):
     Returns:
         trajectory  A list of `ase.Atoms` objects where each element represents
         each step during the relaxation.
-    '''
+    """
     if vasp_flags is None:  # Immutable default
         vasp_flags = VASP_FLAGS.copy()
 
```

Comment:
Make sure you have this change in here - https://github.com/Open-Catalyst-Project/Open-Catalyst-Dataset/pull/61/files#diff-b0cc919ad64652733f261954c39ad3a2db3a6b501eca71a7719a5d4627e760abR182.